lib/src/widgets/helpers/default_status_indicators/first_page_exception_indicator.dart

```diff
@@ -4,15 +4,16 @@ import 'package:flutter/material.dart';
 class FirstPageExceptionIndicator extends StatelessWidget {
   const FirstPageExceptionIndicator({
     required this.title,
-    this.message,
+    required this.message,
     this.onTryAgain,
+    this.tryAgainBtnTxt,
```

Comment:
could this maybe named in a way that fits the other parameter names?
e.g. `tryAgainText`